.github/workflows/application-onboarding-helm-template.yml

```diff
@@ -81,9 +88,23 @@ jobs:
         if: steps.directory_check.outputs.directory_exists == 'false'
         run: cat <<< $(yq -e '(.image.repository)="${{ secrets.HTTPBIN_ECR_REPOSITORY}}"' vsp-infra-application-manifests/apps/${{ inputs.application_name }}/${{ inputs.application_environment }}/values.yaml) > vsp-infra-application-manifests/apps/${{ inputs.application_name }}/${{ inputs.application_environment }}/values.yaml
 
+      - name: Modify image tag to pull from ECR 
+        if: steps.directory_check.outputs.directory_exists == 'false'
+        run: cat <<< $(yq -e '(.image.tag)="${{ env.IMAGE_TAG }}"' vsp-infra-application-manifests/apps/${{ inputs.application_name }}/${{ inputs.application_environment }}/values.yaml) > vsp-infra-application-manifests/apps/${{ inputs.application_name }}/${{ inputs.application_environment }}/values.yaml
+
       - name: Modify ingress environment to use dev
         if: steps.directory_check.outputs.directory_exists == 'false'
-        run: cat <<< $(yq -e '(.ingress.environment)="${{ inputs.application_environment }}"' vsp-infra-application-manifests/apps/om-test/dev/values.yaml) > vsp-infra-application-manifests/apps/om-test/dev/values.yaml
+        run: cat <<< $(yq -e '(.ingress.environment)="${{ inputs.application_environment }}"' vsp-infra-application-manifests/apps/${{ inputs.application_name }}/${{ inputs.application_environment }}/values.yaml) > vsp-infra-application-manifests/apps/${{ inputs.application_name }}/${{ inputs.application_environment }}/values.yaml
+
+      - name: Enable ingress
+        if: steps.directory_check.outputs.directory_exists == 'false'
+        run: cat <<< $(yq -e '(.ingress.enabled)=true' vsp-infra-application-manifests/apps/${{ inputs.application_name }}/${{ inputs.application_environment }}/values.yaml) > vsp-infra-application-manifests/apps/${{ inputs.application_name }}/${{ inputs.application_environment }}/values.yaml
```

Comment:
Why not just make it enabled: true in the template?

Reply:
🤔 Idk lol. Changing it now

Reply:
Done